shared/common-adapters/user-proofs.desktop.js

```diff
@@ -39,57 +75,36 @@ class ProofsRender extends Component {
     }
   }
 
-  _renderProofRow (proof: Proof, idx: number) {
-    const metaBackgroundColor = shared.metaColor(proof)
-    const proofStatusIconType = shared.proofStatusIcon(proof)
-    const proofNameStyle = shared.proofNameStyle(proof)
-    const onClickProfile = () => { this._onClickProfile(proof) }
-
-    const proofStyle = {
-      ...globalStyles.selectable,
-      width: 208,
-      display: 'inline-block',
-      wordBreak: 'break-all',
-      ...styleProofName,
-    }
-
-    const meta = proof.meta && proof.meta !== metaNone && <Meta title={proof.meta} style={{backgroundColor: metaBackgroundColor}} />
-
-    const proofIcon = proofStatusIconType && <Icon type={proofStatusIconType} style={styleStatusIcon} onClick={() => this._onClickProof(proof)} />
-
-    return (
-      <p style={{...styleRow, paddingTop: idx > 0 ? 8 : 0}} key={`${proof.id}${proof.type}`}>
-        <Icon style={styleService} type={shared.iconNameForProof(proof)} hint={proof.type} onClick={onClickProfile} />
-        <span style={styleProofNameSection}>
-          <span style={styleProofNameLabelContainer}>
-            <Text inline={true} className='hover-underline-container' type='Body' onClick={onClickProfile} style={proofStyle}>
-              <Text inline={true} type='Body' className='underline' style={proofNameStyle}>{proof.name}</Text>
-              <Text className='no-underline' inline={true} type='Body' style={styleProofType}><wbr />@{proof.type}<wbr /></Text>
-            </Text>
-            {meta}
-          </span>
-        </span>
-        {proofIcon}
-      </p>
-    )
-  }
-
   render () {
-    const {loading} = this.props
+    const {isLoading} = this.props
+    const pad = idx => idx > 0 ? {marginTop: globalMargins.tiny} : {}
+    const missingProofsRealCSS = `
+      .user-proof-row .user-proof-row__name { text-underline: none; font-weight: normal; }
+      .user-proof-row:hover .user-proof-row__name { text-decoration: underline; font-weight: bold; }
+    `
     return (
-      <div style={{...styleContainer, minHeight: loading ? 120 : 50, ...this.props.style}}>
-        <Box style={{...styleLoading, opacity: loading ? 1 : 0}}>
-          {[147, 77, 117].map((w, i) => <LoadingProofRow key={i} index={i} textBlockWidth={w} />)}
+      <Box style={{...styleContainer(isLoading), ...this.props.style}}>
+        <Box style={{...styleLoading(isLoading)}} className='loading'>
+          {[147, 77, 117].map((w, idx) => <LoadingProofRow key={idx} textBlockWidth={w} style={pad(idx)} />)}
         </Box>
-        <Box style={{...globalStyles.fadeOpacity, opacity: !loading ? 1 : 0}}>
-          {this.props.proofs.map((p, idx) => this._renderProofRow(p, idx))}
+        <Box style={{...styleDoneLoading(isLoading)}} className='notLoading'>
```

Comment:
Oh darn. I'll remove that. I was investigating the overflow issue occurring from a previous PR and was leveraging these class names to identify the culprits.